### PR TITLE
Update pack versions

### PR DESCRIPTION
**Pull request by ritawatson**
This PR updates the versions of packs listed below - please check the docs for packs you updated:

**AWS EFS CSI** - This doc is also refactored to use the new Packs Doc Layout. Please see questions in the Troubleshooting section. It seems the existing information is more usage than Troubleshooting - should we move it to the Usage section? I added new Troubleshooting information for your review.

Kong
Azure Disk CSI
K8s Dashboard
External-DNS
Open Policy Agent
Calico CNI
External Secret Operator

For docs purposes, this PR also adds a shortcut to the Makefile.

**[Preview](https://deploy-preview-1093--docs-spectrocloud.netlify.app/integrations)**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **zulfilee**
You can assign the PR to them by writing `/assign @zulfilee` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**
- **[prow/OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/prow/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-aws-efs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:72]:
```suggestion
- Add EFS as a base CSI layer in a cluster profile.
```

File: content/docs/06-integrations/00-aws-efs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:168]: Why are we showing this IAM policy? We should add an explanation.

File: content/docs/06-integrations/00-aws-efs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:211]: Yes, this is usage information.

File: content/docs/06-integrations/00-aws-efs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:214]: The commands can be used by the section needs clarification and clean-up.

File: content/docs/06-integrations/00-aws-efs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:258]:
```suggestion
  kubectl describe sa efs-csi-controller-sa --namespace kube-system
```
We should always show the long-form of command to help the reader.

File: content/docs/06-integrations/00-aws-efs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:269]:
```suggestion
  kubectl get all --label app.kubernetes.io/name=aws-efs-csi-driver --namespace kube-system
```

File: content/docs/06-integrations/00-aws-efs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:275]:
```suggestion
  sudo mount --types -efs --options tls file-system-dns-name efs-mount-point/
```

File: content/docs/06-integrations/00-aws-efs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-aws-efs.md:349]:
```suggestion
data "spectrocloud_registry" "public_registry" {
  name = "Public Repo"
}

data "spectrocloud_pack_simple" "csi-aws-efs" {
  name    = "csi-aws-efs"
  version = "1.4.0"
  type = "helm"
  registry_uid = data.spectrocloud_registry.public_registry.id
}
```

Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-aws-efs.md:349]: @ritawatson, so the current Terraform code is if you are using this Pack as an add-on to a cluster profile.

When using this Pack as a base layer, you need the following terraform code.

```
data "spectrocloud_registry" "public_registry" {
  name = "Public Repo"
}
data "spectrocloud_pack_simple" "aws-efs" {
  name    = "aws-efs"
  version = "1.4.0"
  type = "helm"
  registry_uid = data.spectrocloud_registry.public_registry.id
}
```

Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-aws-efs.md:349]: Pink is system app (storage layer), orange is addon
CleanShot 2023-01-30 at 19 35 56@2x

Comment (karl-cardenas-coding) [reply content/docs/06-integrations/00-aws-efs.md:349]: I asked in a thread that I tagged you on if this is the plan.

File: content/docs/06-integrations/00-kong.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kong.md:19]:
```suggestion
The Kong integration is an Ingress Controller for Kubernetes that configures ingress with a load balancer. You can use the Kong as an application load balancer for your application.
```
The later statement is confusing so let's just remove it.

File: content/docs/06-integrations/00-kong.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-kong.md:41]:
```suggestion
The integration adds the Kong Ingress Controller, which exposes a Kubernetes service of type LoadBalancer.
```
We use service a bit through our documentation so being specific helps a bit.

**Review by karl-cardenas-coding — COMMENTED** [review]
Overall good, left some feedback about the EFS pack.

File: content/docs/06-integrations/00-longhorn.md
Comment (brathina-spectro) [inline content/docs/06-integrations/00-longhorn.md:25]: Kubernetes version for longhorn should be v1.22 at a minimum. 
The first version of longhorn v1.3.1 is not supported for k8s versions 1.25 and higher.
So, we're rolling out a new version of longhorn v1.4.0. 

This is an example where external dependencies (in this case Kubernetes) vary for a specific version of the pack and the pre-req / limitations should be documented for every version.

**Review by ritawatson — COMMENTED** [review]
Incorporated review feedback.

File: content/docs/06-integrations/00-aws-efs.md
Comment (ritawatson) [reply content/docs/06-integrations/00-aws-efs.md:346]: fixed typo.

File: content/docs/06-integrations/00-aws-efs.md
Comment (ritawatson) [reply content/docs/06-integrations/00-aws-efs.md:23]: Removed.

File: content/docs/06-integrations/00-aws-efs.md
Comment (ritawatson) [reply content/docs/06-integrations/00-aws-efs.md]: Incorporated.

**Comment by ritawatson** [comment]
@brathina-spectro @karl-cardenas-coding This PR is updated with suggested changes. Please take a look and approve if all looks good. 

I refactored Longhorn to use the new Packs layout, as the prerequisite is different between the two versions.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#pullrequestreview-1281965334" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by brathina-spectro — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1093#pullrequestreview-1283562316" title="Approved">brathina-spectro</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1093#pullrequestreview-1281965334" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1093#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
